src/pages/Login.tsx

```diff
@@ -69,15 +83,38 @@ const LoginButton = styled.button`
   height: 2.75rem;
   border: none;
   border-radius: 4px;
-  cursor: pointer;
   color: ${({ theme }) => theme.colors.gray900};
+  cursor: ${({ disabled }) => (disabled ? 'not-allowed' : 'pointer')};
 `;
 
+const validateEmail = (value: string) => {
+  if (!value.trim()) return 'ID는 반드시 입력되어야 합니다.';
+  const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+  return emailRegex.test(value)
+    ? ''
+    : 'ID는 이메일 형식이어야 합니다.';
+};
+
+const validatePassword = (value: string) => {
+  if (!value.trim()) return 'PW는 반드시 입력되어야 합니다.';
+
+  return value.length >= 8
+    ? ''
+    : 'PW는 최소 8글자 이상이어야 합니다.';
```

Comment:
```suggestion
  if (!value.trim()) return 'PW는 반드시 입력되어야 합니다.';
  if (value.length < 8) return 'PW는 최소 8글자 이상이어야 합니다.';
  
  return '';
```

위와 같은 식으로 해주시는게 코드 일관성이 높아 보입니다.